### ISSUE TITLE
[Terminal Preview - Move pane]: Screen reader is not announcing any information when user try to 'Move' tab or 'Move' pane.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#15159]
### Windows Terminal version

1.18.902.0

### Windows build number

25324.1011

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 25324.1011)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

Prequisite:

1. Go to setting using 'Ctrl + ,'.
2. Open 'JSON File'.
3. Add the following JSON in your terminal’s settings.json within the “actions” array
{ "keys": "f1", "command": { "action": "moveTab", "window": "1" }, },

Repro Steps:

1.Open Windows Terminal.
4.Open multiple 'Command prompt' using 'Ctrl + Shift + 1' keys.
5.Now press F1 to perform 'Move' tab functionality using keyboard.
6.Observe the issue.

Users Experience:
Screen reader users will get impacted if screen reader does not provide information about the moving tab or pane.

Guideline link:
https://www.w3.org/TR/WCAG21/#info-and-relationships

Attachments:
[Screen reader is not announcing any information when user try to 'Move' tab or 'Move' pane..zip](https://github.com/microsoft/terminal/files/11197584/Screen.reader.is.not.announcing.any.information.when.user.try.to.Move.tab.or.Move.pane.zip)

### Expected Behavior

Screen reader should announce any information when user try to 'Move' tab or 'Move' pane.

### Actual Behavior

Screen reader is not announcing any information when user try to 'Move' tab or 'Move' pane.

**Comment by carlos-zamora** [comment]
> Screen reader should announce any information when user try to 'Move' tab or 'Move' pane.

This should be easily accomplished using [RaiseNotificationEvent](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisenotificationevent?view=winrt-22621).
